### ISSUE TITLE
Failing to launch dominions.py

### ISSUE THREAD

**Issue by Delnar** [braddockcg/dominions#1]
When launching the game via ssh -p2222 dominions@localhost I get the following error:

Traceback (most recent call last):
  File "/home/dominions/dominions.py", line 16, in <module>
    import filelock
  File "/usr/local/lib/python3.5/dist-packages/filelock/__init__.py", line 21
    __version__: str = version
               ^
SyntaxError: invalid syntax
Connection to localhost closed.

Something is not working with filelock.

**Comment by Delnar** [comment]
So I updated the version of linux in the docker image to 18.04 and that fixed the above problem.  I'm still having issues with somethings not working, but I was able to get past the above error.

**Comment by braddockcg** [comment]
Great!  Do you want to try to get a gang together to play a game?

**Comment by Delnar** [comment]
I don't have a way to host a game externally but would love to play if there is one available.

**Comment by braddockcg** [comment]
@Delnar I can host a game if we find some more players.  I have three friends who might be interested if we can get some more people.  e-mail me at braddock@braddock.com

**Comment by Delnar** [comment]
I have one friend that might be interested.  I'm from a small town and I think in the 90's my BBS was the only one in town with the Dominions game on (it was Dominion 500).  I ran a BBS called Arc of Infinity and my friend I wrote  a map and empire viewer (in Pascal) so we could see the layout of the game.  Good times.  That's why I was interested in your github.  I'm actually thinking about taking what you've done and see if I can get it to work on a Raspberry Pi.  

Also, my copy is loading the Dominion's screen but I'm getting a dos error 2 which is file not found.  Any ideas?  I've seen dos error 3 for path no found and dos error 5 for access denied and have fixed both of those errors.  I'm just not sure what file it's missing.
